src/App.vue

```diff
@@ -255,7 +259,8 @@ export default Vue.extend({
     drawer: false,
     isProjectCreationDialogVisible: false,
     isOrganismCreationDialogVisible: false,
-    isExpanded: true
+    isExpanded: true,
+    hotreloadStatus: undefined
```

Comment:
I don't know if it makes much of a difference but at least in the past we tried to prefer `null` over `undefined` consistently where possible.

Reply:
Oh, right :+1: